### PR TITLE
Fix vendoring and subsequent runtime crashes

### PR DESCRIPTION
**Pull request by thediveo**
- fixes and refactors vendoring and .proto import paths, thus avoiding unwanted module aliasing.
- fixes runtime crashes due to multiple registrations of a module and its alias module(es).
- ensures minimal protobuf version installation dependency to fix crashes due to old protobuf installations not supporting a more recent API package generation (protoc).
- adds API 1.4.
- fixes and upgrades examples.

### CODE REVIEW

**Comment by thediveo** [comment]
BTW, these PR fixes make pycontainerd work on arm64 like a charm 😁. (_except for the compile times when installing grpc as there's no ready binary wheel_)

File: script/genpb2.sh [hunk 1]
```diff
@@ -2,17 +2,17 @@
 #
 # Copyright 2019 Siemens AG
 #
-#   Licensed under the Apache License, Version 2.0 (the "License");
-#   you may not use this file except in compliance with the License.
-#   You may obtain a copy of the License at
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may not
```
Comment (Silvanoc) [inline script/genpb2.sh]: Please, "domesticate" your IDE not to make style changes. Specially to licenses. It simplifies reviews a lot.

File: README.md [hunk 2]
```diff
@@ -1,47 +1,56 @@
...
- * 1.2.1 for the second release for API 1.2
- * 1.3.0 for the first release for API 1.3
+
+- 1.2.1 for the second release for API 1.2
+- 1.3.0 for the first release for API 1.3
 
 ## License
 
-This project is licensed as [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0) (SPDX-License-Identifier: Apache-2.0).
+This project is licensed as [Apache License, Version
```
Comment (Silvanoc) [inline README.md]: Although having a line break inside of a MarkDown hyperlink parentheses (text or URL) [should work](https://stackoverflow.com/questions/19985235/break-long-lines-in-markdown-code), if it's your IDE automatically doing so, please take care not to create line breaks in the wrong place (between `]` and `(`).

**Review by Silvanoc — APPROVED** [review]
I have some minor comments that won't block the PR, but it would be nice if you could consider them. Please let me know if you want to take any action. If not, I'll merge it.

File: script/genpb2.sh (hunk 1 above)
Comment (thediveo) [reply script/genpb2.sh]: that was a case of problem between chair and keyboard, I'll clean this up so the license doesn't get messed up.

File: README.md (hunk 2 above)
Comment (thediveo) [reply README.md]: I'll clean this up too, maybe I can also find a way to keep the markdown reformatter plugin in vsc from stumping on links.

File: script/genpb2.sh (hunk 1 above)
Comment (Silvanoc) [reply script/genpb2.sh]: It wasn't messed up. But being a license, I had to check it more carefully than otherwise :slightly_smiling_face:

**Pull request merged**